lib/commands/gesture.js

```diff
@@ -79,6 +82,13 @@ function isDoubleTap (gestures) {
   return false;
 }
 
+function isTwoFingerTap (gestures) {
+  if (gestures.length === 1 && gestures[0].action.toLowerCase() === 'twofingertap') {
```

Comment:
Hi @sergey-plevako-badoo! Is `twofingertap` in the W3C gesture spec? I don't remember it being there.

If not, we can't just add new actions. I like this change in general but I think this function probably needs to be rewritten to actually detect a 2-digit tap based on the number of digits in the action chain, rather than creating a new action type.

I'm not up on the spec these days though so maybe I can be convinced otherwise. @imurchie should probably also comment.